docs/src/architecture/domain-model.md

```diff
@@ -0,0 +1,198 @@
+# thin-edge Domain Model
+
+The following section introduces the **thin-edge** domain model.
+
+**thin-edge** is the glue between the **cloud** and **devices**. I.e. it interfaces on the one hand **devices** (especially resource constrained industrial **devices**), and connects on the other hand the **cloud**.
+The **cloud** could be some IoT Cloud Platform, as e.g. [Cumulocity](https://www.softwareag.cloud/site/product/cumulocity-iot.html), [Azure](https://azure.microsoft.com) or [AWS](https://aws.amazon.com); or an on-premise "IoT system".
```

Comment:
Just some minor grammar fixes. 

The **cloud** could be some IoT Cloud Platform, such as [Cumulocity](https://www.softwareag.cloud/site/product/cumulocity-iot.html), [Azure](https://azure.microsoft.com), [AWS](https://aws.amazon.com); or an on-premise "IoT system".